docs/changes.rst

```diff
@@ -681,9 +681,9 @@ This is purely a bug fix release:
 * If you use flatmap in a way that the strategy on the right hand side depends
   sensitively on the left hand side you may have occasionally seen Flaky errors
   caused by producing unreliable examples when minimizing a bug. This use case
-  may still be somewhat fraught to be honest. This code is due a major rearchitecture
-  for 1.8, but in the meantime this release fixes the only source of this error that
-  I'm aware of.
+  may still be somewhat fraught to be honest. This code needs is scheduled to be
+  rearchitected for 1.8, but in the meantime this release fixes the only source
```

Comment:
This sentence no longer makes sense. I think it was fine as is – I’m not convinced this is a useful improvement.